### ISSUE TITLE
OnAppResize doesn't work

### ISSUE THREAD

**Issue by ricoberger** [maxence-charriere/go-app#489]
First of all thanks for this awesome project.

I'm currently facing an issue, where I want to use `OnAppResize`  to set the current width of the window to show a div or not.

The problem can be reproduced with the following example. The `width` is never updated:

```go
// +build wasm

package components

import (
	"github.com/maxence-charriere/go-app/v7/pkg/app"
)

type Toolbar struct {
	app.Compo
	width int
}

func (t *Toolbar) OnAppResize(ctx app.Context) {
	// window is resized, but the width isn't updated
	width := app.Window().Get("innerWidth").Int()
	t.update(width)
}

func (t *Toolbar) OnMount(ctx app.Context) {
	// width is set to the initial window width
	width := app.Window().Get("innerWidth").Int()
	t.update(width)
}

func (t *Toolbar) update(width int) {
	app.Dispatch(func() {
		t.width = width
		t.Update()
	})
}

func (t *Toolbar) Render() app.UI {
	return app.Div().Body(
		app.Div().Text(t.width),
		app.If(t.width > 992, app.Div().Text("test 1")),
		app.If(t.width <= 992, app.Div().Text("test 2")),
	)
}
```

Any idea why this isn't working?

**Comment by maxence-charriere** [comment]
What version do you use ?

**Comment by ricoberger** [comment]
Hi @maxence-charriere thank you for the fast answer... just realized that it is available since v7.2.0.

I was using v7.1.2. With v7.2.0 it works perfectly, sorry for the noice.
